cypress/integration/common/index.js

```diff
@@ -51,6 +51,21 @@ Given('user is on landing Genetics Terms page', () => {
 	cy.visit(baseURL);
 });
 
+
+Given('user enters {string} as the URL', (a) => {
+	cy.visit(a);
```

Comment:
This step is a simple navigation ( cy.visit) - we already have a step that does it, so no need to duplicate the code 
Should be deleted